<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnuQuiz</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255,255,255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="questionIndex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AnuQuiz - Quiz Creator (Question x out of x)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="menuButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>-10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255,0,0)</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>New Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,0);
color:white;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Export Quiz</string>
    </property>
   </widget>
   <widget class="QLabel" name="a1Label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A. Answer B</string>
    </property>
   </widget>
   <widget class="QLabel" name="a2Label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B. Answer B</string>
    </property>
   </widget>
   <widget class="QLabel" name="a3Label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C. Answer C</string>
    </property>
   </widget>
   <widget class="QLabel" name="a4Label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>270</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D. Answer C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a1Edit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a2Edit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a3Edit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>210</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="a4Edit">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Question : Default Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="questionEdit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>120</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0,0,255)</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="a2Label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Correct Answer :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="answerSpinbox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>4</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
